.buildkite/pipeline.yml

```diff
@@ -4,6 +4,24 @@ env:
   DOCKER_COMPOSE_VERSION: "1.25.5"
   TERRAFORM_VERSION: "1.6.4"
 
+# This section is used to define the plugins that will be used in the pipeline.
+# See https://buildkite.com/docs/pipelines/integrations/plugins/using#using-yaml-anchors-with-plugins
+common:
+  - test_collector_plugin: &test_collector_plugin
+      test-collector#v1.11.0:
```

Comment:
Bump the version:
- https://github.com/buildkite-plugins/test-collector-buildkite-plugin/releases/tag/v1.11.0